### DISCUSSION TITLE
Forms\Components\Select->default Now Working

### DISCUSSION THREAD

**Discussion by ismaeltj9** [filamentphp/filament#14241] [Help]
### Package

Form builder

### Package Version

v3.2.112

### How can we help you?

Hello all,
The first option is loaded, not default.
```php
Forms\Components\Select::make('tipo_movimentacao')
    ->options([
        '+/-' => '+/-',
        '=' => '=',
    ])
    ->default('=')
    ->selectablePlaceholder(false)
    ->required(),
```
![image](https://github.com/user-attachments/assets/d9f4d9ec-d2df-4017-a2a7-4c9f528e7577)
Is it a bug or I am doing something wrong?

**Comment by leandrocfe** [comment]
If you are using the panel builder, `default` method will work only in the CreatePage.
If you are using only the form builder, make sure you did `$this->form->fill()` in the mount method
